es/03_0_Entendiendo_Su_Configuracion_Bitcoin.md

```diff
@@ -0,0 +1,31 @@
+# Capítulo tres: Comprensión de la configuración de Bitcoin
+
+Ahora está listo para comenzar a trabajar con la interfaz de línea de comandos `bitcoin-cli`. Pero eso primero requiere que comprenda la configuración de Bitcoin y las características de su billetera, que es lo que se explicará en este capítulo.
+
+Para este y futuros capítulos, suponemos que tiene un VPS con Bitcoin instalado, con la ejecución de `bitcoind`. También suponemos que está conectado a testnet, lo que le permite acceder a bitcoins sin utilizar fondos reales. Puede hacer esto con Bitcoin Standup en Linode.com en [§2.1: Configurando un VPS Bitcoin-Core  con Bitcoin Standup](02_1_Configurando_un_VPS_Bitcoin-Core_StackScript.md), o mediante otros medios, en [§2.2: Configuración de una máquina Bitcoin-Core a través de otros medios](02_2_Configurando_Bitcoin_Core_Otros.md).
```

Comment:
- Change `con la ejecucion de bitcoind` to `ejecutando/corriendo bitcoind`.
- Change `[§2.1: Configurando un VPS Bitcoin-Core con Bitcoin Standup]` to `[2.1: Configuración de un Servidor Privado Virtual de Bitcoin-Core con Bitcoin Standup]`. Later we can change it again, but along all chapters in one consistent pull request.
- Suggestion: Open an issue to break all chapter lines to 180 characters, to avoid this kind of long comments on single lines.